gbfs.md

```diff
@@ -207,6 +229,15 @@ stations              | Yes       | Array that contains one object per station i
 \- is_returning        | Yes       | 1/0 boolean - is the station accepting bike returns (if a station is full but would allow a return if it was not full then this value should be 1)
 \- last_reported       | Yes       | Integer POSIX timestamp indicating the last time this station reported its status to the backend
 
+### geofencing_zone_status.json
+Field Name            | Required  | Defines
+--------------------- | ----------| ----------
+geofencing_zones      | Yes       | Array that contains one object per geofencing zone in the system as defined below
+\- geofencing_zone_id  | Yes       | Unique identifier of a geofencing zone (see geofencing_zone_information.json)
+\- num_bikes_available | Yes       | Number of bikes available for rental
+\- num_bikes_disabled  | Optional  | Number of disabled bikes in the geofencing zone. 
+\- is_returning        | Yes       | 1/0 boolean - is the geofencing zone accepting bike returns (business rules, such as city regulation may affect whether we are allowed to return in this zone or not during certain time slots)
```

Comment:
I'd recommend calling this field `accepting_returns`.

Reply:
OK! I mirrored it on the naming already present in the station feed to not have different naming conventions. I didn't like the name either, but figured uniformity was better. Thoughts?